.github/workflows/upgrade-liff-sdk.yml

```diff
@@ -0,0 +1,40 @@
+name: Upgrade LIFF SDK
+
+on:
+  schedule:
+    - cron: '0 0 1 1 0'
+
+jobs:
+  update-dep:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 'lts/*'
+
+      - name: Check and Update dependencies
+        run: |
+          npx -p npm-check-updates ncu -u @line/liff
+          npm install
+
+      - name: Run lint task
```

Comment:
I think the `lint` step is not necessary for this Workflow.

Reply:
I think linting, testing, and building tasks should be separate workflow from `Upgrade LIFF SDK`, and `Upgrade LIFF SDK` should only be used for updating the SDK version and making a PR.